Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -109,6 +110,39 @@ ThemeColorController FindThemeColorController()
             return controllerTransform.GetComponent<ThemeColorController>();
         }
 
+        private void SetTransformSync(Transform transf, ControllerSyncMode syncMode, ControllerSyncMode targetMode)
+        {
+
+            #if UDONSHARP
+
+            transf.GetComponent<UdonBehaviour>().enabled = syncMode < targetMode;
```

Comment:
can you explain this?

Reply:
The enum is an ordering. It lets you exclude _just_ the power and gain from being synced (seems useful to me). So for the gain slider, targetMode is ExcludePowerAndGain. Anything less than that (ie. ExcludePower and All) mean that the gain slider should be synced.

Reply:
i meant the GetComponent line specifically

Reply:
The weird naming of the function and variables?
Idk, but I can try and come up with some more descriptive and accurate names.

Reply:
The UdonBehaviour generalization is to avoid having two functions, one for GlobalSlider and GlobalToggle,
However I can make it two separate functions if desired